core/src/ledger/storage/masp_conversions.rs

```diff
@@ -90,15 +296,49 @@ where
                 denom,
                 wl_storage.storage.block.epoch,
             );
-            current_convs.insert(
-                (addr.clone(), denom),
-                (MaspAmount::from_pair(old_asset, -(reward.1 as i32)).unwrap()
-                    + MaspAmount::from_pair(new_asset, reward.1 as i32)
-                        .unwrap()
-                    + MaspAmount::from_pair(reward_asset, reward.0 as i32)
+
+            if *addr == address::nam() {
+                let new_normed_inflation = (new_normed_inflation
+                    % I256::from(u64::MAX))
+                .try_into()
+                .unwrap();
```

Comment:
Is this unrwap always safe?

The previous code used a lot of unwraps anyways, so it's probably fine